packages/lesswrong/lib/publicSettings.ts

```diff
@@ -81,3 +80,5 @@ export const gatherTownRoomName = new DatabasePublicSetting<string | null>("gath
 // Public elicit settings
 export const elicitSourceURL = new DatabasePublicSetting('elicitSourceURL', 'https://LessWrong.com')
 export const elicitSourceId = new DatabasePublicSetting('elicitSourceId', 'XCjOpumu-')
+
+export const mapboxAPIKeySetting = new DatabasePublicSetting<string | null>('mapbox.apiKey', null) // API Key for the mapbox map and tile requests
```

Comment:
Why refactor this to here?

Reply:
I initially refactored this to accommodate a different implementation, but actually I think it makes more sense here. `PetrovDayButton` uses it, which is totally unrelated to `CommunityMap` where the setting was initially stored.